civis/ml/_model.py

```diff
@@ -819,6 +859,15 @@ def train(self, df=None, csv_path=None, table_name=None,
         polling_interval : float, optional
             Check for job completion every this number of seconds.
             Do not set if using the notifications endpoint.
+        validation_data : str, optional
+            Source for validation data. There are currently two options:
+            `'train'` (the default), which uses training data for validation;
```

Comment:
"uses training data" --> "cross-validates over training data"? CivisML doesn't construct metrics with in-sample predictions.